code/script.js

```diff
@@ -219,107 +222,172 @@ const generateBoard = () => {
         </div>
       </div>
     `
-  })
-}
+  });
+};
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
 
 // This function to start (and restart) the game
 const start = () => {
+  // Showing the game board when you restart the game.
+  board.style.display = 'flex';
+
+  // Hiding the win or lose page when you restart the game.
+  winOrLose.style.display = 'none';
+
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
-  // What else should happen when we start the game?
-}
+
+  // The computer chooses a character which the player has to guess.
+  setSecret();
+
+  // Invoking the function generateBoard to generate the board on start.
+  generateBoard();
+
+  // Select the default question as the starting question.
+  selectQuestion();
```

Comment:
Nice catch!  The first select <change> event is not registered on 'brown hair', thus firing the function in the start function is a good way to circumvent the bug.